dandi/upload.py

```diff
@@ -712,22 +713,54 @@ def process_path(path, relpath):
 
             extant = client.get_asset_bypath(ds_identifier, "draft", str(relpath))
```

Comment:
coverage reports that none of this code is covered... codecov bug?

Reply:
There was a window where the Docker tests were failing because the code hadn't been updated for https://github.com/dandi/dandi-api/pull/144.  If you checked the coverage during that window, the code that's only run under Docker wouldn't have shown up as covered.